component-samples/scripts/cbor2bytes.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# USAGE:
+#     ./cbor2bytes rvinfo_body
+#
+# Example of single RV information:
+#
+#   ./cbor2bytes.sh "[[[5, \"localhost\"], [3, 8040], [12, 1], [2, h'7F000001'], [4, 8443]]]"
+#   NOTE: here 7F000001 is the hex representation of the IP address 127(7F).0(00).0(00).1(01)
+#
+# Example of multiple RV instructions:
+#     ./cbor2bytes.sh  "[[[5, \"localhost\"], [3, 8040], [12, 1], [2, h'7F000001'], [4, 8443]],
+#                   [[5, \"localhost\"], [3, 8040], [12, 1], [2, h'6A140101'], [4, 8443]]]"
+
+cbor=$1
```

Comment:
Provide a default input too (could be the default RVInfo that we use). If no inputs, use that and generate the output.

nitpick: The input is actually in diagnostic format, the output would be a CBOR representation. You can rename the variable appropriately.

Reply:
Done.

Reply:
Done.